### PR TITLE
Hunter 2.11

### PR DESCRIPTION
**Pull request by isaachier**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -70,17 +69,17 @@ if (UNIX)
...
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/bin)
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/lib)
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/lib)
+set(JANSSON_TEMP_DIR ${CMAKE_CURRENT_BINARY_DIR}/tmp)
 
 # Give the debug version a different postfix for windows,
 # so both the debug and release version can be built in the
 # same build-tree on Windows (MSVC).
-if (WIN32)
+if (WIN32 AND NOT CMAKE_DEBUG_POSTFIX)
```
Comment (isaachier) [inline CMakeLists.txt:80]: Not sure if I need this but I know Hunter uses `"d"` for `CMAKE_DEBUG_POSTFIX` and I didn't want this to break it.

File: CMakeLists.txt [hunk 2]
```diff
@@ -607,80 +606,67 @@ set(VERSION     ${JANSSON_DISPLAY_VERSION})
...
+          EXPORT janssonTargets
+          LIBRARY DESTINATION "lib"
+          ARCHIVE DESTINATION "lib"
+          RUNTIME DESTINATION "bin"
+          INCLUDES DESTINATION "include")
+
+  install(FILES ${JANSSON_HDR_PUBLIC}
+          DESTINATION "include")
+
+  # Install the pkg-config.
```
Comment (isaachier) [inline CMakeLists.txt:654]: As of now, don't feel the need to potentially patch this file. Seeing as Hunter won't use it, didn't bother looking into it.

File: CMakeLists.txt (hunk 1 above)
Comment (ruslo) [reply CMakeLists.txt:80]: Should not be a problem. Imported target will hide real file name and location. Hunter set `*_POSTFIX` to non-empty value so such configuration like Debug will not overwrite files installed by Release configuration.

**Review by ruslo — APPROVED** [review]

**Comment by ruslo** [comment]
Released: https://github.com/hunter-packages/jansson/releases/tag/v2.11-p0

Please send this patches upstream, they are not related to hunterization.

**Comment by isaachier** [comment]
Great thanks. Yes I plan on it.

**Pull request merged**